Doc/library/re.rst

```diff
@@ -1378,6 +1378,27 @@ when there is no match, you can test whether there was a match with a simple
    if match:
        process(match)
 
+Match objects are proper :class:`~collections.abc.Sequence` types. You can access
```

Comment:
This is not true with this PR, Sequence has a number of other requirements (e.g. an `index` and `count` method).

Reply:
Nice catch, thanks! I added `index` and `count` by adapting the implementation of `tuple.index` and `tuple.count`. I also updated the unit test to cover all `Sequence` mixin methods.